### PR TITLE
Added ISO88595 Code page support

### PR DESCRIPTION
**Pull request by wOvAN**
Hi I've added ISO88595 support, test text in txt files in \pdu\pdutext\test_text

### CODE REVIEW

File: smpp/pdu/pdutext/iso88595_test.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+package pdutext
+
+import (
+	"bytes"
+	"io/ioutil"
+	"testing"
+)
+
+var (
+	ISO88595_Bytes []byte = readBytesFromFile("test_text/iso88595_test.txt")
```
Comment (fiorix) [inline smpp/pdu/pdutext/iso88595_test.go]: can we put these test files in a `testdata` directory instead?

File: smpp/pdu/pdutext/iso88595_test.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	Utf8_Bytes     []byte = readBytesFromFile("test_text/iso88595_test_utf8.txt")
+)
+
+func TestISO88595Encoder(t *testing.T) {
+	want := []byte(ISO88595_Bytes)
+	text := []byte(Utf8_Bytes)
+	s := ISO88595(text)
+	if s.Type() != 0x06 {
+		t.Fatalf("Unexpected data type; want 0x03, have %d", s.Type())
```
Comment (fiorix) [inline smpp/pdu/pdutext/iso88595_test.go]: want 0x06

File: smpp/pdu/pdutext/iso88595_test.go [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+		t.Fatalf("Unexpected text; want %q, have %q", want, have)
+	}
+}
+
+func TestISO88595Decoder(t *testing.T) {
+	want := []byte(Utf8_Bytes)
+	text := []byte(ISO88595_Bytes)
+	s := ISO88595(text)
+	if s.Type() != 0x06 {
+		t.Fatalf("Unexpected data type; want 0x03, have %d", s.Type())
```
Comment (fiorix) [inline smpp/pdu/pdutext/iso88595_test.go]: ditto

File: smpp/pdu/pdutext/iso88595_test.go [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+	}
+	have := s.Decode()
+	if !bytes.Equal(want, have) {
+		t.Fatalf("Unexpected text; want %q, have %q", want, have)
+	}
+}
+
+func readBytesFromFile(aFileName string) []byte {
```
Comment (fiorix) [inline smpp/pdu/pdutext/iso88595_test.go]: can we call this parameter just `filename`? doesn't really matter but i'd prefer it that way

File: smpp/pdu/pdutext/iso88595_test.go [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+	dat, err := ioutil.ReadFile(aFileName)
+	if err != nil {
+		return nil
```
Comment (fiorix) [inline smpp/pdu/pdutext/iso88595_test.go]: given that this is a test, it should actually `panic()` if the test data file does not exist

**Comment by wOvAN** [comment]
Made the changes, Except Travis CI

**Comment by fiorix** [comment]
To fix travis you'll have to update [.travis.yml](https://github.com/fiorix/go-smpp/blob/master/.travis.yml) and have only Go 1.7, 1.8 and tip.

**Comment by fiorix** [comment]
On a side note, I believe `golint` will complain about your use of underscores in variable names: `iso88595UTF8_Bytes` -> `iso88595UTF8Bytes`

**Comment by wOvAN** [comment]
Underscores fixed, about transmitter_test I'm not ready to troubleshoot this

**Comment by fiorix** [comment]
Ok. Will keep this open until I find time to fix the tests and then merge your PR.

**Comment by nezorflame** [comment]
This is fixable by moving to the new context. I guess it's time.

**Comment by nezorflame** [comment]
see https://github.com/fiorix/go-smpp/pull/50

**Comment by fiorix** [comment]
Merged in #50. There's one follow up though: https://github.com/fiorix/go-smpp/commit/24f8c599fed71be1a83b304ec861d4230fd3df94#diff-6830008c9742aaabdd083a6e0433abffR255

**Pull request closed without merging**